Directory.Build.props

```diff
@@ -1,10 +1,11 @@
 <Project>
   <PropertyGroup>
-    <DotnetVersion>net6.0</DotnetVersion>
+    <DefaultTargetDotnetVersion>net8.0</DefaultTargetDotnetVersion>
+    <TargetDotnetVersions>net6.0;$(DefaultTargetDotnetVersion)</TargetDotnetVersions>
     <Platforms>x64;ARM64</Platforms>
     <AssemblyVersion>2.0.0.0</AssemblyVersion>
     <FileVersion>2.0.0.0</FileVersion>
-    <Version>2.120.2</Version>
+    <Version>2.120.3</Version>
```

Comment:
Just noticed it now, but maybe a relevant change worth it to bump the minor version

Reply:
well the minor is reserved for the cef version, well we used rev .100 for avalonia 11